apps/admin-gui/src/app/shared/components/consent-hubs-list/consent-hubs-list.component.ts

```diff
@@ -52,6 +56,9 @@ export class ConsentHubsListComponent implements OnChanges {
   ngOnChanges(): void {
     this.dataSource = new MatTableDataSource<ConsentHub>(this.consentHubs);
     this.setDataSource();
+    this.selection.changed.subscribe(() => {
+      this.selectionChanged.emit();
+    });
```

Comment:
please look on how we are doing selection in other tables, I don't think that the selection changed is needed (eg. members-list)